### ISSUE TITLE
how to run it? which end is up?

### ISSUE THREAD

**Issue by dckc** [rchain/rbalance#1]
`sbt compile` seems to work
`sbt run` doesn't seem to do anything.
`Main` seems to consist of `IO{ ExitCode.Success }`

How is one supposed to invoke this code?

**Comment by cia-spy** [comment]
@dckc 
looks as if @leithaus  has not wired up main yet, important, but a small detail.
you may get the reportS from 
```
sbt console
import coop.rchain.rbalance.txns._
PithiaClique.reportAdjustments()
BarcelonaClique.reportAdjustments()
```

**Comment by dckc** [comment]
that doesn't quite work:

```
scala> PithiaClique.reportAdjustments()
<console>:12: error: not found: value PithiaClique
       PithiaClique.reportAdjustments()
       ^
```

But I can get a bit closer; this seems to compute for a while:

```
scala> import coop.rchain.rbalance.txns._
import coop.rchain.rbalance.txns._

scala> RHOCTxnGraphClosure.PithiaClique.reportAdjustments()

```

but then I get:

```
java.io.FileNotFoundException: src/main/resources/wallets_9371743_withzeros.txt (No such file or directory)
```

**Comment by cia-spy** [comment]
@dckc that file is under resources folder.
i dont have the time now, if @leithaus doenst get to it by tomorrow, i'll put a fix for it as this is a trivial fix

**Comment by dckc** [comment]
In e741834 I added calls to `reportAdjustments` from `Main`. It's included in PR #5 .
